cmake/onnxruntime_unittests.cmake

```diff
@@ -693,7 +693,7 @@ if (onnxruntime_USE_ROCM)
   target_include_directories(onnxruntime_test_all PRIVATE  ${onnxruntime_ROCM_HOME}/hipfft/include ${onnxruntime_ROCM_HOME}/include ${onnxruntime_ROCM_HOME}/hiprand/include ${onnxruntime_ROCM_HOME}/rocrand/include ${CMAKE_CURRENT_BINARY_DIR}/amdgpu/onnxruntime ${CMAKE_CURRENT_BINARY_DIR}/amdgpu/orttraining)
 endif()
 if (onnxruntime_ENABLE_TRAINING_TORCH_INTEROP)
-  target_link_libraries(onnxruntime_test_all PRIVATE Python::Module)
+  target_link_libraries(onnxruntime_test_all PRIVATE Python::Python)
```

Comment:
I remember @snnn mentioned to me to change all `Python::Python` to `Python::Module` and all symbols will be resolved magically. Do you see build failures without this change?